### PR TITLE
[update] Tipue-Search v7.1.0

### PR DESCRIPTION
**Pull request by nkr0**
Update library to v7.1.0. GitHub repository deleted. Thus, auto-update doesn't work anymore.

### CODE REVIEW

**Comment by nkr1729** [comment]
When I added the files I followed the template of previous versions and uploaded all the files. However, `tipuesearch_content.js` is only an example file that came with the package and is not required.

**Comment by nkr1729** [comment]
hey, can we merge this?

File: ajax/libs/Tipue-Search/package.json [hunk 1]
```diff
@@ -7,27 +7,11 @@
...
   ],
-  "version": "5.0.0",
+  "version": "7.1.0",
   "author": {
     "name": "Tipue",
     "url": "http://www.tipue.com"
   },
   "license": "MIT",
-  "homepage": "http://www.tipue.com/search/",
-  "autoupdate": {
```
Comment (zackbloom) [inline ajax/libs/Tipue-Search/package.json]: Out of curiosity, why did you decide to disable auto-updating?

Comment (MattIPv4) [reply ajax/libs/Tipue-Search/package.json]: The repo no longer exists it seems, I would assume that's why :)

Comment (nkr0) [reply ajax/libs/Tipue-Search/package.json]: Yep. I couldn't find any links on the [website](http://www.tipue.com/search/) pointing to a repository. Only a download link to this version is available. The package is still under MIT license.

**Comment by nkr0** [comment]
Just a reminder to merge. Or are we waiting for any more changes?

**Comment by MattIPv4** [comment]
Hi there,

We are no longer accepting pull requests that manually add versioned files for libraries. Instead, please ensure that the library has an up-to-date auto-update config for either NPM or Git in the cdnjs repository.

You can find out more about our auto-update config structure in the [CONTRIBUTING](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) and [CONTRIBUTING-WIP](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md) documents.

**Pull request closed without merging**